src/Test/PerformanceTesting.csproj

```diff
@@ -98,6 +98,9 @@
     <Content Include="Perf\tests\helloworld\hello_world.csx">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="Perf\infra\MonitorShareTask.bat">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Do we need to copy this to output?  I assume we're not going to run from output, because this script is what gets the output directory on to the machine (chicken and egg problem).  I would assume we'll need to copy this to the machine somewhere and update it manually as needed.